README.md

```diff
@@ -20,11 +20,25 @@ $ git clone --recursive https://github.com/pytorch/test-infra
 │       ├── download.pytorch.org
 │       └── metrics.pytorch.org
 ├── stats                                # CI related stats commited automatically by a bot
-└── tools                                # Tools and scripts
-    ├── clang-tidy-checks
-    └── scripts
+├── tools                                # Tools and scripts
+|   ├── clang-tidy-checks
+|   └── scripts
+└── torchci                              # Code for hud.pytorch.org and our pytorch bots which run there   
+    └── pages 
 ```
 
+## Setting up your Dev environment to locally run hud.pytorch.org
+1. Install yarn:
+    `brew install yarn`
```

Comment:
Probably need to mention that this is for MacOS.

Reply:
good point